contracts/adapters/MigrationExtension.sol

```diff
@@ -81,13 +83,15 @@ contract AaveMigrationExtension is BaseExtension, FlashLoanSimpleReceiverBase, I
     ITradeModule public immutable tradeModule;
     IDebtIssuanceModule public immutable issuanceModule;
     INonfungiblePositionManager public immutable nonfungiblePositionManager;
+    IMorpho public constant MORPHO = IMorpho(0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb);
+    IBalancerVault public constant BALANCER = IBalancerVault(0xBA12222222228d8Ba445958a75a0704d566BF2C8);
```

Comment:
for consistency, should we move other constructor inputs to this type of initialization?

Reply:
Actually went ahead and turned these two addresses into immutable values that can be specified in the constructor.

That way we can use it on l2 / other networks where the addresses might differ.